assets/model_monitoring/components/src/shared_utilities/df_utils.py

```diff
@@ -6,8 +6,11 @@
 
 data_type_double_group = ["float", "double"]
 data_type_long_group = ["long", "int", "bigint", "short"]
+data_type_numerical_group = ["float", "double", "decimal"]
+data_type_categorical_group = ["string", "boolean", "timestamp", "date", "binary"]
```

Comment:
why we want to treat timestamp, binary as categorical columns by default? Any guideline?

Reply:
I checked with PM. According to PM: 

"It depends a bit on the customer scenario. For timestamp, we can default to categorical because it may shift a lot, if it is smooth it may make more sense for it to be numerical. For binary it’s by definition categorical"